runtime/compiler/optimizer/VarHandleTransformer.cpp

```diff
@@ -245,6 +242,11 @@ int32_t TR_VarHandleTransformer::perform()
                   TR::Node::recreate(callTree->getNode(), newOpCode);
                }
 
+
+            // Anchoring all the children for varhandle
+            anchorAllChildren(node, tt);
+            performTransformation(comp(), "%sVarHandle access methods found, working on node %p\n", optDetailString(), node);
```

Comment:
If this line is moving anyway, can it be changed to `dumpOptDetails`? The transformation is happening unconditionally